src/hu_stuff.c

```diff
@@ -1153,21 +1153,42 @@ static void HU_widget_build_sttime(void)
   int offset = 0;
   extern int time_scale;
 
-  if (time_scale != 100)
+  if (gameaction == ga_completed)
+    plr->btuse_tics = 0;
+
+  if (hud_level_time & HUD_WIDGET_HUD ||
```

Comment:
Since the entire build function is only called if one of these conditions turn true, I think it is sufficient to merely check for `hud_level_time` here.

Reply:
I'll try that later.

Reply:
Nope, it's not enough: if `hud_level_time == HUD_WIDGET_HUD`, both the Level time *and* Use time are displayed on the Automap when pressing the Use key, when only the latter should be shown.

Reply:
Turns out that these conditions were insufficient too, as they suffered the `hud_level_time == HUD_WIDGET_HUD`. I guess we need another `automapactive`; added in e389dc025fca01db54b2d89b0fe4a7519f255e04.

Reply:
Hm, does this work?

```patch
--- a/src/hu_stuff.c
+++ b/src/hu_stuff.c
@@ -1151,8 +1151,7 @@ static void HU_widget_build_sttime(void)
   int offset = 0;
   extern int time_scale;
 
-  if ((hud_level_time & HUD_WIDGET_HUD     && !automapactive) ||
-      (hud_level_time & HUD_WIDGET_AUTOMAP &&  automapactive))
+  if (hud_level_time || !plr->btuse_tics)
   {
     if (time_scale != 100)
     {
```

Reply:
> Hm, does this work?

No, I think it doesn't. Please merge!

Reply:
Seems to work properly. Want me to apply it?

Reply:
> > Hm, does this work?
> 
> No, I think it doesn't. Please merge!

Once again, I read your comment late. What makes you think it doesn't work?

Reply:
It shows total level times in addition to use time.

Reply:
Right, I didn't test that.